From 49ad0583ea39d27ea7e5f65262465feb58b0cfd2 Mon Sep 17 00:00:00 2001
From: Athenas Jimenez Gonzalez <athenas.jimenez.gonzalez@intel.com>
Date: Wed, 30 Mar 2016 16:32:58 +0000
Subject: [PATCH] Fix gemspec

---
 rubygem-jeremymcanally-pending.gemspec | 1 -
 1 file changed, 1 deletion(-)

diff --git a/rubygem-jeremymcanally-pending.gemspec b/rubygem-jeremymcanally-pending.gemspec
index 88de383..e4486cc 100644
--- a/rubygem-jeremymcanally-pending.gemspec
+++ b/rubygem-jeremymcanally-pending.gemspec
@@ -10,7 +10,6 @@ Gem::Specification.new do |s|
   s.description = "pending lets you define a block of test code that is currently \"pending\" functionality, similar to RSpec's pending method.".freeze
   s.email = "jeremymcanally@gmail.com".freeze
   s.extra_rdoc_files = ["History.txt".freeze, "README.txt".freeze]
-  s.files = ["History.txt".freeze, "Manifest.txt".freeze, "README.txt".freeze, "Rakefile".freeze, "lib/pending.rb".freeze, "tasks/ann.rake".freeze, "tasks/bones.rake".freeze, "tasks/gem.rake".freeze, "tasks/git.rake".freeze, "tasks/manifest.rake".freeze, "tasks/notes.rake".freeze, "tasks/post_load.rake".freeze, "tasks/rdoc.rake".freeze, "tasks/rubyforge.rake".freeze, "tasks/setup.rb".freeze, "tasks/spec.rake".freeze, "tasks/svn.rake".freeze, "tasks/test.rake".freeze, "test/test_pending.rb".freeze]
   s.homepage = "http://jeremymcanally.com".freeze
   s.rdoc_options = ["--main".freeze, "README.txt".freeze]
   s.rubyforge_project = "pending".freeze
-- 
2.7.4

